src/com/projectkorra/projectkorra/firebending/WallOfFire.java

```diff
@@ -170,13 +171,10 @@ private void display() {
 
 	private void initializeBlocks() {
 		Vector direction = this.player.getEyeLocation().getDirection();
-		direction = direction.normalize();

```

Comment:
Any reason they are no longer normalized?

Reply:
> Any reason they are no longer normalized?

Player's locations retunred with getLocation and getEyeLocations already have normalized direction vectors. They are normalized by default.